### PR TITLE
QUEST_JOBSTEP.tsv // 2098-2137

### PR DESCRIPTION
**Pull request by zaxxer**
those are mainly the same repeating for all class masters. initial
translation was differing a bit between class masters but the context is
also the same (prove your skill)..

tryed to remove some word repition
added articles
put monster types into plural
and changed some mere word groups to full sentences - reads less half
assed imo.

theres still many classes to go here

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
 QUEST_JOBSTEP_20150317_002095	Hand over the corpses to Necromancer Master
 QUEST_JOBSTEP_20150317_002096	You've collected some parts of the corpses. Go back to Necromancer.
 QUEST_JOBSTEP_20150317_002097	Obtain %s
-QUEST_JOBSTEP_20150317_002098	Go to Swordsman Master in Klaipeda.
+QUEST_JOBSTEP_20150317_002098	Go to the Swordsman Master in Klaipeda.
 QUEST_JOBSTEP_20150317_002099	Defeat Honeypins at Deep Indigo Forest
-QUEST_JOBSTEP_20150317_002100	Swordsman Master is emphasizing the importance of training with actual fights. Defeat Honeypins at Deep Indigo Forest and accumulate the experiences.
-QUEST_JOBSTEP_20150317_002101	Defeated Honeypins. Report to Swordsman Master.
-QUEST_JOBSTEP_20150317_002102	Go to Peltasta Master in Klaipeda.
+QUEST_JOBSTEP_20150317_002100	The Swordsman Master is emphasizing the importance of training in actual combat. Defeat Honeypins at Deep Indigo Forest and gather combat experience.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Honeypins -> the Honeypin

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002101	You defeated enough Honeypins. Report back to the Swordsman Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You defeated enough Honeypins -> You defeated Honeypin.

File: QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002102	Go to the Peltasta Master in Klaipeda.
 QUEST_JOBSTEP_20150317_002103	Defeat Gaigalas in Flowerless Forest
-QUEST_JOBSTEP_20150317_002104	Peltasta Master wants you to prove your skills. Defeat Gaigalas in Flowerless Forest and prove your skills.
-QUEST_JOBSTEP_20150317_002105	Defeated Gaigalas. Report to Peltasta Master.
+QUEST_JOBSTEP_20150317_002104	The Peltasta Master wants you to prove your skills. Defeat Gaigalas in Flowerless Forest to demonstrate your skills.
+QUEST_JOBSTEP_20150317_002105	You defeated enough Gaigalas. Report back to the Peltasta Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You defeated enough -> You defeated

File: QUEST_JOBSTEP.tsv [hunk 4]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
-QUEST_JOBSTEP_20150317_002130	Cursoir Master wants you to prove your skills. Defeat Honeypins in Deep Indigo Forest to prove your skills.
-QUEST_JOBSTEP_20150317_002131	Report to Corsair Master
-QUEST_JOBSTEP_20150317_002132	Defeated Honeypins. Report to Corsair Master.
-QUEST_JOBSTEP_20150317_002133	Talk to Doppelsoeldner Master
-QUEST_JOBSTEP_20150317_002134	Go meet Doppelsoeldner Master in Forgotten Port.
-QUEST_JOBSTEP_20150317_002135	Doppelsoeldner Master wants you to prove your skills by defeating Gaigalas in Flowerless Forest.
-QUEST_JOBSTEP_20150317_002136	Report to Doppelsoeldner Master
-QUEST_JOBSTEP_20150317_002137	Defeated Gaigalas. Report to Doppelsoeldner Master.
+QUEST_JOBSTEP_20150317_002107	Go find the Highlander Master in Klaipeda.
+QUEST_JOBSTEP_20150317_002108	The Highlander Master wants you to gather more experience. He asks you to defeat Honeypins at Deep Indigo Forest.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: defeat Honeypins -> defeat Honeypin

File: QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
-QUEST_JOBSTEP_20150317_002131	Report to Corsair Master
-QUEST_JOBSTEP_20150317_002132	Defeated Honeypins. Report to Corsair Master.
-QUEST_JOBSTEP_20150317_002133	Talk to Doppelsoeldner Master
-QUEST_JOBSTEP_20150317_002134	Go meet Doppelsoeldner Master in Forgotten Port.
-QUEST_JOBSTEP_20150317_002135	Doppelsoeldner Master wants you to prove your skills by defeating Gaigalas in Flowerless Forest.
-QUEST_JOBSTEP_20150317_002136	Report to Doppelsoeldner Master
-QUEST_JOBSTEP_20150317_002137	Defeated Gaigalas. Report to Doppelsoeldner Master.
+QUEST_JOBSTEP_20150317_002107	Go find the Highlander Master in Klaipeda.
+QUEST_JOBSTEP_20150317_002108	The Highlander Master wants you to gather more experience. He asks you to defeat Honeypins at Deep Indigo Forest. 
+QUEST_JOBSTEP_20150317_002109	You defeated enough Honeypins. Report back to the Highlander Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You defeated enough Honeypins -> You defeated Honeypin

File: QUEST_JOBSTEP.tsv [hunk 6]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002110	The Hoplite Master possesses the skill which his colleagues can rely on. Defeat Collimencias in Forest of Incremental Rocks to prove your skills.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Collimencias -> Colimencia

File: QUEST_JOBSTEP.tsv [hunk 7]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002111	You defeated enough Colimencias. Report back to the Hoplite Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You defeated enough Colimencias -> You defeated Collimencia

File: QUEST_JOBSTEP.tsv [hunk 8]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002112	Go to the Barbarian Master in Siauliai Eastern Woods.
+QUEST_JOBSTEP_20150317_002113	Meet the Barbarian Master in Siauliai Eastern Woods
+QUEST_JOBSTEP_20150317_002114	The Barbarian Master wants other people to respect him for defeating Poatas that are known for attacking people. Defeat Poatas in Forest of Incremental Rocks.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: The Barbarian Master wants other people to respect him for defeating Poatas -> The Barbarian Master wants other people to respect him by defeating the Poata

Defeat Poatas -> Defeat Poata

File: QUEST_JOBSTEP.tsv [hunk 9]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002115	You Defeated enough Poatas. Report back to the Barbarian Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You defeated enough Poatas -> You defeated Poata

File: QUEST_JOBSTEP.tsv [hunk 10]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002116	Meet the Rodelero Master in Fedimian.
+QUEST_JOBSTEP_20150317_002117	The Rodelero Master wants you to gather experience in real combat. Defeat Collimencias in Forest of Incremental Rocks to prove your skills.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Collimencias -> Colimencia

File: QUEST_JOBSTEP.tsv [hunk 11]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002118	You Defeated enough Colimencias. Report back to the Rodeleo Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You Defeated enough Colimencias -> You defeated Colimencia.

File: QUEST_JOBSTEP.tsv [hunk 12]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002119	Meet the Cataphract Master in High Crooked Road.
+QUEST_JOBSTEP_20150317_002120	The Cataphract Master tells you that it is important to break anything that disturbes your way. Defeat Gaigalas in Flowerless Forest to prove your skills.
+QUEST_JOBSTEP_20150317_002121	You Defeated enough Gaigalas. Report to the Cataphract Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You Defeated enough Gaigalas -> You defeated Gaigalas. 

Report to the Cataphract Master. > Report back to the Cataphract Master.

File: QUEST_JOBSTEP.tsv [hunk 13]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002122	Go to the Squire Master in Fedimian.
+QUEST_JOBSTEP_20150317_002123	Defeat Poatas and find the Symbol of Legwyn Family
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Defeat Poatas -> Defeat Poata

File: QUEST_JOBSTEP.tsv [hunk 14]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002124	The Squire Master wants the Symbol of Legwyn Family. Defeat Poatas in Forest of Incremental Rocks to find the symbol.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Defeat Poatas -> Defeat the Poata

File: QUEST_JOBSTEP.tsv [hunk 15]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002125	Report back to the Squire Master
+QUEST_JOBSTEP_20150317_002126	You found the Ornament of Legwyin Family. Report back to the Squire Master.
+QUEST_JOBSTEP_20150317_002127	Retrieve the Ornament of Legwyin Family
+QUEST_JOBSTEP_20150317_002128	Talk to the Corsair Master
+QUEST_JOBSTEP_20150317_002129	Meet the Squire Master in Starving Demon's Road
+QUEST_JOBSTEP_20150317_002130	The Corsair Master wants you to prove your skills. Defeat Honeypins in Deep Indigo Forest to demonstrate your skills.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Defeat Honeypin -> Defeat Honeypin

File: QUEST_JOBSTEP.tsv [hunk 16]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002131	Report back to the Corsair Master
+QUEST_JOBSTEP_20150317_002132	You defeated enough Honeypins. Report back to the Corsair Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You defeated enough Honeypins -> You defeated Honeypin.

File: QUEST_JOBSTEP.tsv [hunk 17]
```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
...
+QUEST_JOBSTEP_20150317_002133	Talk to the Doppelsoeldner Master
+QUEST_JOBSTEP_20150317_002134	Meet the Doppelsoeldner Master in Forgotten Port.
+QUEST_JOBSTEP_20150317_002135	The Doppelsoeldner Master wants you to prove your skills by defeating Gaigalas in Flowerless Forest. 
+QUEST_JOBSTEP_20150317_002136	Report back to the Doppelsoeldner Master
+QUEST_JOBSTEP_20150317_002137	You defeated enough Gaigalas. Report back to the Doppelsoeldner Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: You defeated enough Gaigalas -> You defeated Gaigalas.

**Comment by Sunaries** [comment]
From looking at the Korean lines, I can tell that they're talking about one specific **boss monster** (Gaigalas (_This mob is actually called Gaigalas, and not Gaigala)_, Poata, Colimencia, Honeypin), so I assume that you wouldn't have to defeat more than one of the boss mob. You can also see that in each line there are different Class Masters asking to defeat the boss monster.

**Comment by zaxxer** [comment]
^after i read Gaigalas as plural, not as a singular name, i thought it would be the objective to defeat multiple enemies of one type in all those class quests. oh well, i hope i got everything fixed here. ill also reread the whole thing later again.

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**